### ISSUE TITLE
Feature Request [toggle  radar when in combat]

### ISSUE THREAD

**Issue by pribnisckyjr** [syndenbock/Farmer#15]
Option to hide radar when in combat. Would be nice for it to hide while fighting then come back when combat is done

**Comment by stamarty** [comment]
agreed. came here specifically to make this request, as well as the request to turn on radar automatically when mounting and turn it off when not mounted
